### PR TITLE
Switch from Radix v3 to v4 

### PR DESCRIPTION
**Pull request by Zlaticanin**
Update Radix client to v4:
- v3 is only accepting bug fixes at this point
- v4 has more polished API, full RESP3 support, support for context.Context

### CODE REVIEW

File: connection_producer.go [hunk 1]
```diff
@@ -2,16 +2,15 @@ package redis
 
 import (
 	"context"
-//	"crypto/x509"
-//	"encoding/base64"
+	//	"crypto/x509"
+	//	"encoding/base64"
```
Comment (robmonte) [inline connection_producer.go:6]: Looks like some leftover commented out code

File: redis.go [hunk 2]
```diff
@@ -152,12 +152,12 @@ func newUser(ctx context.Context, db *radix.Pool, username string, req dbplugin.
 	aclargs = append(aclargs, args...)
 	fmt.Printf("Appended args: %v\n", aclargs)
 	var response string
-	
+
 	//	err = db.Do(radix.Cmd(&response, "ACL", "SETUSER", username, "on", ">" + req.Password, args...))
-	err = db.Do(radix.Cmd(&response, "ACL", aclargs...))
+	err = db.Do(ctx, radix.Cmd(&response, "ACL", aclargs...))
```
Comment (robmonte) [inline redis.go:157]: Same as above, a leftover commented out line of code

**Review by robmonte — APPROVED** [review]
Looks good to me 👍

**Review by maxcoulombe — APPROVED** [review]

**Review by austingebauer — APPROVED** [review]
👍

**Pull request merged**